source/Stargate-HealthCheck/CriticalHealth.class.st

```diff
@@ -33,6 +36,12 @@ CriticalHealth >> initializeDetailedBy: aDetailsObject onHealthCheckNamed: aName
 	concern := aName
 ]
 
+{ #category : #testing }
+CriticalHealth >> isNotCritical [
```

Comment:
It's weird to implement a method with a negative behavior.
It might be clearer to implement isCritical and then let the user negate it.